src/components/organisms/ChatSidebar/ChatSidebar.scss

```diff
@@ -20,6 +22,12 @@ $header-height: 45px;
     transition: transform 0.3s;
     background-color: rgba($saturated-black, 0.9);
     box-shadow: -10px 0 40px rgba(0, 0, 0, 0.5);
+
+    // change position to relative -> it starts its calculation from container top and it should be top: 0
+    @media (min-width: $large-laptop-width) {
```

Comment:
I'm wondering whether instead of these media queries/breakpoints, if we should just define these overrides as a sidebar variant (eg. `&--pinned` or similar). That way we would have more control as to when we enable/disable it (eg. in venue config, or as a button that the user can click on the sidebar itself to 'pin it open', etc) That might satisfy @Sparkle-T's concerns + also allow us to put more control over the UX into the end user's hands.

(cc // @mike-lvov @mcflyok FYI: what do you think?)

Reply:
made as `pinned` variant.